src/slic3r/GUI/Tab.cpp

```diff
@@ -1970,6 +1970,14 @@ void TabPrint::build()
         optgroup->append_single_option_line("only_one_wall_first_layer");
         optgroup->append_single_option_line("reduce_crossing_wall");
         optgroup->append_single_option_line("max_travel_detour_distance");
+        optgroup->append_single_option_line("small_area_infill_flow_compensation");
```

Comment:
Would it be better to put this option the "Small Area Infill Flow Compensation" group too?

Reply:
Moved it over, but it has the downside that the full width text input box no longer works. Input feels very cramped that way but doesn't have any annoying line breaks, so should be ok.